hive/src/main/java/com/netease/arctic/hive/utils/HiveMetaSynchronizer.java

```diff
@@ -222,6 +228,207 @@ public static void syncHiveDataToArctic(SupportHive table, HMSClientPool hiveCli
     }
   }
 
+  /**
+   * Synchronize the data change of the arctic table to hive table
+   * @param table support hive table
+   */
+  public static void syncArcticDataToHive(SupportHive table) {
+    UnkeyedTable baseStore;
+    if (table.isKeyedTable()) {
```

Comment:
Should this method be in HiveDataSynchronizer ?

Is there some unit test for this method?

Reply:
I don't think we have a class named `HiveDataSynchronizer` now, and the `HiveMetaSynchronizer` contains both methods `syncHiveSchemaToArctic` and `syncHiveDataToArctic`, which means `Schema` and `Data` are both kinds of `Meta`.

The unit test class is in the module `ams/server` which is `TestHiveCommitSync`, I will move it to module `hive`.

Reply:
Is the integration test under the ams/server module? Should additional unit tests be added specifically for this method?

Reply:
Not the integration test, it‘s a specialized test class for syncing Amoro data to Hive.